examples/code/role_playing.py

```diff
@@ -0,0 +1,79 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from colorama import Fore
+
+from camel.societies import RolePlaying
+from camel.typing import TaskType
+from camel.utils import print_text_animated
+
+
+def main(model_type=None) -> None:
+    task_prompt = "Develop a poll app"
+    language = "JavaScript"
+    domain = "Sociology"
+    meta_dict = {"language": language, "domain": domain}
+    role_play_session = RolePlaying(
+        f"{language} Programmer",
```

Comment:
I think this in-place prompt engineering should not be here. The cool thing about camel is that it incapsulate all the prompts. So the external API can only enter the parameters and all the prompt engineering happens behind the API. This is for the future rework.

Reply:
Sounds good